linux_os/guide/services/ntp/service_ntp_enabled/rule.yml

```diff
@@ -22,7 +22,7 @@ severity: high
 references:
     anssi: NT012(R03)
     disa: "160"
-    nist: AU-8(1)
+    nist: AU-8(1),AU-12(1)
```

Comment:
```suggestion
    nist: CM-6(a),CM-6(b)
```

Reply:
Updating to CM is out of scope of this PR.